src/components/Navigation/OnlyMine/constant.ts

```diff
@@ -0,0 +1,37 @@
+export const filterByDefault = [
```

Comment:
**nit:** // usually comment mark as "**nit:**" at the beginning - can be treated as optional change
I would prefer the default setting to leave in `src/basics/LocalCache/defaultConfig.ts` or be in a separate file located in `src/basics/LocalCache/`. In this case I would suggest to use filter as 
```
{
selectAll : true,
onlyMyWorkflow: true, 
...
}
```
In this case labels will leave in a separate string files in OnlyMine folder. String file will look similar to: `src/components/Executions/Tables/WorkflowExecutionTable/strings.ts` and can be used as `t('inputOutputTooltip')`

We just starting moving to string files. It's a good practice to put all your strings in a separate locations, as it's simplifies localization in future.

This change is fully optional, as the pattern you used is all over out code at this point.

Reply:
moved the config to localcache folder, and added the string file in onlymine folder